Eastern-Research-Group/mywaterway
# feature/apply updated prettier formatting

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3168777

## Main Changes:
* Formats most of the .js files. We recently updated to Prettier 1.19.1 which comes with some new formatting for React files. 

## Steps to test
1. Run the app locally just to make sure nothing broke, but this branch only contains formatting changes.
